cmd/loadtest/loadtest.go

```diff
@@ -22,7 +23,15 @@ func RunLoadTestCmdF(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
-	newSimpleController := func(id int, status chan<- control.UserStatus) control.UserController {
+	if ok, err := config.IsValid(); !ok {
```

Comment:
```suggestion
	if _, err := config.IsValid(); err != nil {
```

If we are going to return `error` (which is always false if error) simply checking `err` would be more idiomatic. That said, do we need `bool, error` return type for `func (c *Config) IsValid() (bool, error) `? Don't think so, it never returns `true, error` or `false, nil`. See [implementation](https://github.com/streamer45/mattermost-load-test-ng/blob/ae82536590d6956f2c75b4bea8f4269a3cc19205/loadtest/config.go#L27).

Reply:
This pattern is actually shared by all config validation methods in this repo. I am wondering if @agnivade might have more info about this particular case.

If we decide to change I propose we do it in a separate PR that addresses all cases at once.

Reply:
I think I agree with @isacikgoz here. Almost all occurences of "isValid", "valid", "checkValid" in the standard library either return true/false if it does not need to distinguish between the type of error returned, or just error if it needs to distinguish.


One such example of the latter is: `crypto/x509/verify.go:func (c *Certificate) isValid`.

Let's do it in a separate PR.